_1327/main/forms.py

```diff
@@ -104,7 +105,9 @@ def prepare_initial_data(cls, groups, content_type, obj=None):
 				group_permissions = get_perms(group, obj)
 			else:
 				group_permissions = [permission.codename for permission in group.permissions.filter(content_type=content_type)]
-			group_permissions = filter(lambda x: any(permission[0] in x for permission in MenuItem.used_permissions), group_permissions)
+			content_type = ContentType.objects.get_for_model(obj)
+			group_permissions = ["{app}.{codename}".format(app=content_type.app_label, codename=codename) for codename in group_permissions]
+			group_permissions = filter(lambda x: any(permission[0] in x for permission in MenuItem.used_permissions()), group_permissions)
```

Comment:
it takes all permissions each group has from `group_permissions` and filters them to only show permissions  returned from used_permissions